### DISCUSSION TITLE
Control the border-size

### DISCUSSION THREAD

**Discussion by jhortale** [dracula/dracula-ui#76] [Ideas]
It could be great if we have a prop to control the border size, mainly for the inputs.

**Comment by appoks** [comment]
I totally agreed

**Comment by nettofarah** [comment]
I'm down with that too.
Is this something you would like to take a stab on?
If not, we'll make sure to add it to our backlog.

**Comment by jhortale** [comment]
Yeah, definitely I could try to do that,

I'll create an issue with this feature and I'll start to work on it.

**Comment by jhortale** [comment]
What do you think would be better, creating a new prop **border size** or creating some new outline variations with different sizes?

**Reply by nettofarah** [reply]
I think something like `borderSize={draculaSizeScaleValue}` would probably be the most scalable option.

anything beyond that could be customized with inline styles or vanilla css.

WDYT?

**Reply by jhortale** [reply]
totally agree
